### PR TITLE
Add "sems->seems, stems, semis, sens, seams," correction

### PR DESCRIPTION
**Pull request by cfi-gb**
See e.g. https://grep.app/search?q=it%20sems%20&words=true

### CODE REVIEW

**Review by TheGiraffe3 — COMMENTED** [review]
"SEMS" is an abbreviation for "scanning electron microscope", so it may not be necessarily wrong. But it will be the majority of the time...

**Comment by cfi-gb** [comment]
Some recent related comments from e.g. #2858 or #2804

> Any (or almost any) typo could be seen as a valid abbreviation.

or

> Don't worry, any typo could be a name (of a person, of a place, of a fictive character or place). We wouldn't have a dictionary of possible typos if we were to take that into account.

I can remember @peternewman writing something similar in the past but can't find that anymore.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -52588,6 +52588,7 @@ sempaphores->semaphores
 semphore->semaphore
 semphores->semaphores
 sempphore->semaphore
+sems->seems, stems, semis, sens,
```
Comment (TheGiraffe3) [inline codespell_lib/data/dictionary.txt:52591]: Alright, that makes sense. Just one more suggestion.
```suggestion
sems->seems, sees, stems, semis, sens,
```

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:52591]: Nice, thanks a lot for the additional suggestion :+1:

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:52591]: And perhaps `seams` as well.

Comment (TheGiraffe3) [reply codespell_lib/data/dictionary.txt:52591]: Wow, lots of words sems could be a typo of.

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:52591]: Added as well, thanks a lot again for the suggestions :+1:

**Review by TheGiraffe3 — APPROVED** [review]

**Pull request merged**